### ISSUE TITLE
deno_core hello_word.rs not working

### ISSUE THREAD

**Issue by gufoe** [denoland/deno#10632]
Using:
`deno_core = "0.87.0"`

Running the [hello world](https://github.com/denoland/rusty_v8/blob/main/examples/hello_world.rs)

```
thread 'main' panicked at 'assertion failed: prev.is_none()', /home/jake/.cargo/registry/src/github.com-1ecc6299db9ec823/deno_core-0.87.0/ops.rs:137:5
stack backtrace:
   0: rust_begin_unwind
             at /rustc/5c029265465301fe9cb3960ce2a5da6c99b8dcf2/library/std/src/panicking.rs:493:5
   1: core::panicking::panic_fmt
             at /rustc/5c029265465301fe9cb3960ce2a5da6c99b8dcf2/library/core/src/panicking.rs:92:14
   2: core::panicking::panic
             at /rustc/5c029265465301fe9cb3960ce2a5da6c99b8dcf2/library/core/src/panicking.rs:50:5
   3: deno_core::ops::OpTable::register_op
             at /home/jake/.cargo/registry/src/github.com-1ecc6299db9ec823/deno_core-0.87.0/ops.rs:137:5
   4: deno_core::runtime::JsRuntime::register_op
             at /home/jake/.cargo/registry/src/github.com-1ecc6299db9ec823/deno_core-0.87.0/runtime.rs:524:5
   5: jsint::main
             at ./src/main.rs:25:3
   6: core::ops::function::FnOnce::call_once
             at /home/jake/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
```

**Comment by bartlomieju** [comment]
It seems you are registering the same op multiple times; please provide full reproduction.

**Comment by CGQAQ** [comment]
![image](https://user-images.githubusercontent.com/15936231/118355787-9e4c5a00-b5a4-11eb-8af3-0e724d401a1d.png)

repreduced  with [hello world](https://github.com/denoland/rusty_v8/blob/main/examples/hello_world.rs)

**Comment by CGQAQ** [comment]
Oh! I see, It is because [custom op in the example](https://github.com/denoland/deno/blob/24a2e1ef82cf1a081a9b4603a597eb0085755e36/core/examples/hello_world.rs#L26) is duplicated with [the builtin one](https://github.com/denoland/deno/blob/24a2e1ef82cf1a081a9b4603a597eb0085755e36/core/ops_builtin.rs#L20), I'll fix it

**Comment by AaronO** [comment]
I submitted a fix and cleanup in #10645, I had noticed that issue a few weeks back and should have submitted a PR then ...
